docs/monitors/elasticsearch.md

```diff
@@ -122,7 +122,7 @@ monitors:
 
 For more information on the built-in content we have for Elasticsearch,
 [see
-here](https://github.com/signalfx/integrations/tree/master/elasticsearch)
+here](https://github.com/signalfx/integrations/tree/master/collectd-elasticsearch)
```

Comment:
This is a reversion back to the old directory.  See https://github.com/signalfx/integrations/pull/686

Reply:
> This is a reversion back to the old directory. See [signalfx/integrations#686](https://github.com/signalfx/integrations/pull/686)

Will revert when signalfx/integrations#686 is merged.